pkg/securitypolicy/policy.rego

```diff
@@ -21,4 +21,6 @@ plan9_unmount := data.framework.plan9_unmount
 get_properties := data.framework.get_properties
 dump_stacks := data.framework.dump_stacks
 runtime_logging := data.framework.runtime_logging
+scratch_mount := data.framework.scratch_mount
+scratch_unmount := data.framework.scratch_mount
```

Comment:
data.framework.scratch_unmount?

Reply:
hmm... so my tests didn't cover this. need to update them.